flatdata-rs/lib/src/generator.rs

```diff
@@ -0,0 +1,80 @@
+use std::io::Write;
+use std::path::Path;
+use std::path::PathBuf;
+
+/// A helper function wrapping the flatdata generator.
+///
+/// Can be used to write build.rs build scripts, generating outputs either from a single schema,
+/// or recursively from a folder of schemas.
+///
+/// # Examples
+///
+/// `build.rs`
+/// ``` ignore
+/// use std::path::PathBuf;
+///
+/// fn main() {
+///     let out_path = PathBuf::from(std::env::var("OUT_DIR").unwrap());
+///     let test_cases_path = PathBuf::from("assets/schemas");
```

Comment:
We should say that only "*.flatdata" files are consider as schemas.

Reply:
Done